catalogs/usfimr-s1/ingest_s1_evalscript.js

```diff
@@ -0,0 +1,27 @@
+//VERSION=3
```

Comment:
Wooo you can write some js, read it into python script and it works.

I'd hate to have to maintain that multi-escaped single string nightmare it becomes.